### PR TITLE
Added role field to DataOceanUser

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -43,6 +43,12 @@ def create_superuser(self, email, password, **extra_fields):
 
 
 class DataOceanUser(AbstractUser):
+    PROJECT_INITIATOR = 'project_initiator'
+    PROJECT_PARTICIPANT = 'project_participant'
+    ROLES = [
+        ('PROJECT_INITIATOR', 'project_initiator'),
+        ('PROJECT_PARTICIPANT', 'project_participant'),
+    ]
```
Comment (roman-tiukh) [inline users/models.py:51]: 1. owner
2. member

---

Display names may be more readable...
project_initiator -> Owner
project_participant -> Member

-------------------------------------------------
`('PROJECT_INITIATOR', 'project_initiator'),` - here you dont pass variable, you pass simple string...

File: users/models.py [hunk 2]
```diff
@@ -55,6 +61,7 @@ class DataOceanUser(AbstractUser):
         default=settings.LANGUAGE_CODE,
         blank=True,
     )
+    role = models.CharField(choices=ROLES, max_length=20, default='PROJECT_PARTICIPANT')
```
Comment (roman-tiukh) [inline users/models.py:64]: `default='PROJECT_PARTICIPANT'` -> `default=PROJECT_PARTICIPANT`

File: users/models.py (hunk 1 above)
Comment (roman-tiukh) [inline users/models.py:51]: read documentation about choices here pls
https://docs.djangoproject.com/en/3.1/ref/models/fields/#choices

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
not in DataOceanUser...
User can be in many projects with different roles...
Do this in User2Project (m2m) model

File: users/models.py (hunk 1 above)
Comment (IegorSoboliev) [reply users/models.py]: done, thank you!

File: users/models.py (hunk 2 above)
Comment (IegorSoboliev) [reply users/models.py]: Fixed

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: payment_system/models.py [hunk 3]
```diff
@@ -5,6 +5,23 @@
...
+    PARTICIPANT = 'participant'
+    ROLES = [
+        ('INITIATOR', 'initiator'),
+        ('PARTICIPANT', 'participant'),
+    ]
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
+                             related_name='user_projects')
+    project = models.ForeignKey('Project', on_delete=models.CASCADE, related_name='user_projects')
+    role = models.CharField(choices=ROLES, max_length=20, default=PARTICIPANT)
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply payment_system/models.py:19]: Do you think, we can have a blank role?

[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply payment_system/models.py:19]: added

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**